src/pages/Register.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import register from '../assets/styles/pages/register.css'
+import {FaUserAlt} from "react-icons/fa";
+import {FaEnvelope} from "react-icons/fa";
+import {FaKey} from "react-icons/fa";
+
+class Student_Tutor extends React.Component {
+    state = {
+      register:'tutor',
+    }
+    handleStudent = () => {
+        this.setState({ register: 'student' })
+    }
+    handleTutor = () => {
+        this.setState({ register: 'tutor' })
+      }
+    render() {
+        const rol={
+        student:<RegisterStudent/>,
+        tutor:<RegisterTutor/>,
+    }
+    const {student,tutor}=rol;
+    let stateActive=rol[this.state.register];
+    return (
+        <div className="radios">
+            <h2 className="tittle-rol">Do you want to be student or tutor?</h2>
+            <h2 className="rol"><input type="radio" checked={stateActive===student ? true:false} onClick={this.handleStudent}/>student</h2>
```

Comment:
use one handle to change register variable, something like handleRole and add input attributes like name to get value. Is better to use a select instead of radio button.